.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do you really need to add this change?
Or is it a change on your side you did not mean to commit?